addons/website_slides_survey/models/slide_channel.py

```diff
@@ -1,10 +1,50 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models
+from odoo import api, fields, models, _
+from odoo.osv import expression
 
+class ChannelUsersRelation(models.Model):
+    _inherit = 'slide.channel.partner'
+
+    certification_passed = fields.Boolean('Certified', compute='_compute_certification_passed', store=True)
```

Comment:
A wise man once asked me something like 
> Is it tested or could it be premature optimization?

Could we try not storing `certification_passed` either (as it is computed and not the result of succeeding) (it should just need an extra `_search` method).

We should be able to ask for a test on our database to assess the performance cost.

Reply:
Adding  a  _search_passed_method that:
- can be used by `slide.channel` to compute the its number of certified partners
- filter properly the `slide.channe.partner` tree view scope to a given channel (id)
- filter properly the list of all attendees (same view but without `channel_id` in domain)
- (unlike the previous _search method) do not use context but method arguments to determine the use case